src/Swift.Runtime/src/Swift/AnyType.cs

```diff
@@ -21,4 +21,35 @@ public AnyType(SwiftHandle payload)
         _payload = payload;
     }
     public SwiftHandle Payload => _payload;
+
+    /// <summary>
+    /// Creates a new SwiftOptional from a Swift payload
+    /// </summary>
+    static ISwiftObject ISwiftObject.NewFromPayload(IntPtr payload)
+    {
+        return new AnyType(new SwiftHandle(payload));
+    }
+
+    /// <summary>
+    /// Marshals this object to a Swift destination
+    /// </summary>
+    /// <param name="swiftDest"></param>
+    /// <returns></returns>
+    IntPtr ISwiftObject.MarshalToSwift(IntPtr swiftDest)
+    {
+        var metadata = SwiftObjectHelper<AnyType>.GetTypeMetadata();
+        if (!metadata.IsValid)
+        {
+            throw new InvalidOperationException("Cannot marshal AnyType to Swift without metadata");
```

Comment:
Should these strings be localized?

Reply:
Maybe. I put the exceptions in as an afterthought since it looks like this is not an actual valid type because of the lack of a valid TypeMetadata. @kotlarmilos - is `AnyType` a placeholder for the future since it isn't valid?

Reply:
Yes -- it is safe to assume the type is not valid.